auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -2,7 +2,29 @@
   <div id="affiliated-entity-section">
     <v-card flat>
       <div class="table-header">
-        <label><strong>My List </strong>({{ entityCount }})</label>
+        <v-row
+        no-gutters
+        class="mb-n8"
+        >
+          <v-col class="pt-3 pl-1">
+            <label><strong>My List </strong>({{ entityCount }})</label>
+          </v-col>
+          <v-col class="column-actions">
+            <v-select
+            dense multiple
+            class="column-selector pb-2 pr-2"
+            background-color="white"
+            label="Columns to Show"
+            v-model="selectedColumns"
+            :items="columns"
+            :menu-props="{ bottom: true, offsetY: true }"
+            @change="updateHeaders"
```

Comment:
I like to see an explicit function call here, eg, `@change="updateHeaders($event)"`.

Also, why not call `getHeaders` directly here?